### ISSUE TITLE
Term 2, Lesson 16 (Semi-supervised), notebooks and lesson video 5

### ISSUE THREAD

**Issue by paulc00** [udacity/aind-issue-reports#173]
Second layer of discriminator() is defined with
relu2 = tf.maximum(alpha * x2, x2)
and is therefore not using batch normalization

relu2 should use 'bn2' not 'x2':
relu2 = tf.maximum(alpha * bn2, bn2)

The code is incorrect in both the exercise nb and the solution nb as well as the video showing these notebooks.

**Comment by paulc00** [comment]
https://github.com/udacity/deep-learning/tree/master/semi-supervised

**Comment by paulc00** [comment]
Same code, same function:
x7 = tf.layers.conv2d(relu5, 2 * size_mult, 3, strides=1, padding='valid')

should be:
x7 = tf.layers.conv2d(relu6, 2 * size_mult, 3, strides=1, padding='valid')

**Comment by paulc00** [comment]
The above change to x7 will cause the model to crash. I commented out x6 and left x7 referring to relu5 in my copy of the notebook and that model ran OK.
